app/src/internal/java/com/duckduckgo/app/dev/settings/InternalUAInterceptor.kt

```diff
@@ -46,6 +46,7 @@ class InternalUAInterceptor @Inject constructor(
             UAOverride.CHROME -> userAgent.replace(applicationComponent, "").replace(version, "")
             UAOverride.FIREFOX -> "Mozilla/5.0 (Android 11; Mobile; rv:94.0) Gecko/94.0 Firefox/94.0"
             UAOverride.WEBVIEW -> defaultUserAgent.get()
+            UAOverride.NO_APP_ID_AND_NO_VERSION -> userAgent.replace(applicationComponent, "").replace(version, "")
```

Comment:
This is exactly the same as using the CHROME override?

Reply:
You are 100% correct. I have changed the PR and just amended the string for chrome to specify it means not using the app id and the version 👍